apps/payments/next/app/[locale]/[offeringId]/[interval]/landing/page.tsx

```diff
@@ -49,27 +49,28 @@ function reportError(message: string, details?: any) {
  * This needs to be a route handler, since the `signIn` server
  * action needs to be executed from a route handler.
  */
```

Comment:
Could this be changed back to a Route Handler instead of a Page. I just saw this note, and reminded me why it needs to be a route. If not, then you'll see an error as shown below.

```16|payments-next  | tA [Error]: Cookies can only be modified in a Server Action or Route Handler. Read more: https://nextjs.org/docs/app/api-reference/functions/cookies#cookiessetname-value-options```